app/views/cuisines/_cuisine.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="col-md-4 col-sm-6">
+  <div class="cuisine-item">
+    <% if cuisine.available? %>
+      <%= link_to cuisine_path(cuisine.slug), class: "cuisine-link" do %>
+        <%= image_tag cuisine.image, class: "cuisine-image" %>
+        <div class="sentence">
+          <h3><%= cuisine.name %> <span><%= number_to_currency(cuisine.price) %></span></h3>
+          <h6><%= cuisine.category.name %></h6>
+          <p><%= cuisine.description %></p>
+        </div>
+      <% end %>
+    <% else %>
+      <div class="cuisine-not-available">
+      <%= link_to cuisine_path(cuisine.slug), class: "cuisine-link" do %>
+        <%= image_tag cuisine.image, class: "cuisine-image-not-available" %>
```

Comment:
chỉ khác mỗi css của image và div thì tìm hiểu toán tử 3 ngôi chứ đừng tách hẵng if else lặp code thế này nhé bé